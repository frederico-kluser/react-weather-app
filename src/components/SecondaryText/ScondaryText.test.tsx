import renderer from 'react-test-renderer';
import ScondaryText from '.';

test('ScondaryText Renderer', () => {
  const component = renderer.create(
    <ScondaryText text="test" />,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});