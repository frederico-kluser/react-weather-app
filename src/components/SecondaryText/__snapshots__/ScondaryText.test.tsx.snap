// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScondaryText Renderer 1`] = `
<div
  className="sc-bdvvtL jkdmfi"
>
  test
</div>
`;
